app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Send me reset password instructions" %>
```

Comment:
- [ ] After clicking the button to request a link to reset my password, I am taken to this error page.. Kindly inspect it and fix it.

  ![image](https://github.com/c00p75/blog-rails/assets/81468675/f60b872a-b8ec-4e28-9753-cc3ac489a4c5)